src/dishka/async_container.py

```diff
@@ -176,21 +176,31 @@ async def _get(self, key: DependencyKey) -> Any:
             return await self._get_unlocked(key)
 
     async def _get_unlocked(self, key: DependencyKey) -> Any:
-        if key in self._cache:
-            return self._cache[key]
-        compiled = self.registry.get_compiled_async(key)
+        normalized_type = DependencyKey(
```

Comment:
please, do not do anything here. It will be called on each object, so we will slow down everything. All preparations must be done preliminary on container build.